src/scribe_data/cli/list.py

```diff
@@ -0,0 +1,153 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Union
+
+# Load language metadata from JSON file
+METADATA_FILE = Path(__file__).parent.parent / 'resources' / 'language_meta_data.json'
+
+def load_language_metadata() -> Dict:
```

Comment:
I think we're also fine to just use the `with` statement directly without the function :)